From e8b732e61ef29c2b2aceee491068068cd5e70f67 Mon Sep 17 00:00:00 2001
From: Zhang Ying-22455 <ying.zhang22455@nxp.com>
Date: Fri, 25 May 2018 11:34:51 +0800
Subject: [PATCH 468/741] arm64: dts: ls208xa: Add the identify of the platform
 to support to set rcpm bit

commit fc02957f6c721929ec1f5b37b077821578a485ee from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Add the identify of the platform to support set the rcpm.

Signed-off-by: Zhang Ying-22455 <ying.zhang22455@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi
index 4f906d3b8642..65beb52bae7d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi
@@ -839,8 +839,10 @@
 		};
 		
 		ftm0: ftm0@2800000 {
-			compatible = "fsl,ftm-alarm";
-			reg = <0x0 0x2800000 0x0 0x10000>;
+			compatible = "fsl,ls208xa-ftm-alarm";
+                        reg = <0x0 0x2800000 0x0 0x10000>,
+                              <0x0 0x1e34050 0x0 0x4>;
+                        reg-names = "ftm", "pmctrl";
 			interrupts = <0 44 4>;
 		};
 	};
-- 
2.17.1

